src/ImportDefinitionsBundle/Interpreter/AssetsUrlInterpreter.php

```diff
@@ -25,12 +25,15 @@ class AssetsUrlInterpreter extends AssetUrlInterpreter
      */
     public function interpret(Concrete $object, $value, Mapping $map, $data, DefinitionInterface $definition, $params, $configuration)
     {
-        $asset = parent::interpret($object, $value, $map, $data, $definition, $params, $configuration);
+        $assets = [];
```

Comment:
can you also add something like

```
if (is_array($value)) {
for(...)
 ....

return $assetsArray;
}

return [parent::..];
```

just to be BC compatible

Reply:
But this is BC compatible? You get back either an array of assets or `null`?

Reply:
Which is exactly what you got before. Or am I missing something?

Reply:
yes, you are right, I missed the ```(array) $value```

Reply:
NP. :+1: